Sources/MapboxCoreNavigation/BundleAdditions.swift

```diff
@@ -33,7 +33,11 @@ extension Bundle {
      The Mapbox Core Navigation framework bundle.
      */
     public class var mapboxCoreNavigation: Bundle {
+        #if SWIFT_PACKAGE
+        return .module
+        #else
         return Bundle(for: RouteController.self)
+        #endif
```

Comment:
This change is fine, but I’m a little concerned about `mapboxNavigationIfInstalled` below. I don’t think there’s anything like `Bundle.module` for discovering a _different_ module’s bundle.

Reply:
I think I'm getting crash in the same place when starting active guidance navigation.

Reply:
Looks like force unwrapping shouldn't be used when accessing Bundle.mapboxCoreNavigation. E.g. this one crashes as well: 
```
let sdkVersion: String = String(describing: Bundle.mapboxCoreNavigation.object(forInfoDictionaryKey: "CFBundleShortVersionString")!)
```